coordinator/tests/services/translation/demographics.spec.ts

```diff
@@ -51,22 +38,14 @@ describe("convertName fills correct fields only", () => {
     expect(result.suffix).toEqual([{_text: ""}])
   })
 
-  test("usual should return USUAL", () => {
-    const fhirName = {"use": "usual"}
-    const result = demographics.convertName(fhirName)
-    expect(result._attributes).toEqual({use: core.NameUse.USUAL})
-  })
-
-  test("official should return USUAL", () => {
-    const fhirName = {"use": "official"}
-    const result = demographics.convertName(fhirName)
-    expect(result._attributes).toEqual({use: core.NameUse.USUAL})
-  })
+  const cases = [["usual", core.NameUse.USUAL],
```

Comment:
Just because i think it might look neater, could we format these arrays like this:
```
const cases = [
  ["one", 1],
  ["two", 2]
]
```